### ISSUE TITLE
$ENV:BHPSModulePath is deprecated and will be removed July 1st, 2017

### ISSUE THREAD

**Issue by markwragg** [RamblingCookieMonster/PSDeploy#83]
When testing your code in AppVeyor I received the following warning:
```
$ENV:BHPSModulePath is deprecated and will be removed July 1st, 2017
ACTION REQUIRED: Please replace $ENV:BHPSModulePath with $ENV:BHModulePath wherever you use it
```

Currently your deploy.psdeploy.ps1 file uses the deprecated variable and needs updating.

**Comment by markwragg** [comment]
https://github.com/RamblingCookieMonster/PSDeploy/pull/84 raised to address.

**Comment by RamblingCookieMonster** [comment]
Closed in #84 - thanks again!
